code/datums/wires/airlock.dm

```diff
@@ -95,6 +96,8 @@
 
 /datum/wires/airlock/on_cut(wire, mend)
 	var/obj/machinery/door/airlock/A = holder
+	if(isliving(usr) && A.hasPower())	
+		A.shock(usr, 100) //Cutting wires directly on powered doors without protection is not advised. 
```

Comment:
this feels like big fuck you to the players
not to mention doubling the damage is 🤨

Reply:
Damage isn't doubled: 50 was an RNG chance, I made it guaranteed.